modules/react-native-status/android/src/main/java/im/status/ethereum/module/AccountManager.java

```diff
@@ -0,0 +1,345 @@
+package im.status.ethereum.module;
```

Comment:
I was wondering if these files can be grouped into sub-directories

Reply:
Indeed they could be.

I was thinking we would have to update directory structure from : 
`modules/react-native-status/android/src/main/**java**/im/status/ethereum/module/...` to
`modules/react-native-status/android/src/main/**kotlin**/im/status/ethereum/module/...` when https://github.com/status-im/status-mobile/issues/18310 is complete.

We can then have a discussion on the folder structure and when we reach a consensus on folder structure we can then open a new PR for this.